### ISSUE TITLE
Still seeing "this function takes 4 arguments but 2 arguments were supplied" errors in Vulkan CI

### ISSUE THREAD

**Issue by oddhack** [ash-rs/ash#962]
ISTR talking about this but can't find the issue now - sorry if this is a dup. The problem with new attributes is gone - thanks! - but we're still seeing errors in ash-generate (gitlab) / [build-test ](https://github.com/KhronosGroup/Vulkan-Docs/actions/runs/11716916066/job/33140012851#step:5:289)(Github) of form

> error[E0061]: this function takes 4 arguments but 2 arguments were supplied
289  
>   --> ash/src/extensions/amdx/shader_enqueue.rs:77:9

They seem to be confined to the one corresponding AMDX extension, which is interestingly weird.

**Comment by MarijnS95** [comment]
Hey yes, I just put an update in the latest issue that you filed, at https://github.com/ash-rs/ash/issues/959#issuecomment-2482782238:

> After merging and re-running the CI code generation is once again successful, but compiling still fails because of [#343 (comment)](https://github.com/ash-rs/ash/issues/343#issuecomment-2396499207), which I hope to solve entirely with #952 or at the very least fix in #951.

I probably need to discuss with @Ralith how we should design + release a fix for this in the near future. Perhaps I might just merge #951 to `master` and that should fix your immediate build issues.

That immediately preps us for the next breaking release, so I might be branching off at that point in case we want to aim for a nonbreaking release before this point.

**Comment by oddhack** [comment]
@MarijnS95 this seems to be taking quite a while, and I'm inclined to remove ash from the Vulkan spec repo CI until it's resolved - it's not doing any good to raise warnings in every branch for reasons outside the spec repo itself, and it's causing some confusion with Vulkan-Hpp as those project owners keep checking the CI logs to see if the CI warning is coming from their binding or not. The idea was to detect problems with the language bindings before publishing spec updates but it is not fulfilling that purpose, at present.

(Sorry for the dup comment, put it on the wrong issue at first).

**Comment by MarijnS95** [comment]
@oddhack yes please, I've been thinking about doing the same thing. Don't forget to remove it from both GitHub and GitLab.
